Support to SSL connection

Current version of db2 presto plugin does not support SSL connection.


Updated on June 17, 2019, 2:14PM:
Here is the settings I had for the plugin. To enable the ssl, I used the url with sslConnection set to true.
```
connector.name=db2
connection-url=jdbc:db2://db2whoc-flex-performance-riubnzs.services.dal.bluemix.net:50001/BLUDB:sslConnection=true
connection-user=myuser
connection-password=mypassword
```

And here is the error when I am running queries from Presto to access DB2 table:
```
java.sql.SQLException: Query failed (#20190617_180905_00000_jas3k): [jcc][10165][10051][4.25.13] Invalid database URL syntax: jdbc:db2://db2whoc-flex-performance-riubnzs.services.dal.bluemix.net:50001/BLUDB:sslConnection=true. ERRORCODE=-4461, SQLSTATE=42815
	at io.prestosql.jdbc.PrestoResultSet.resultsException(PrestoResultSet.java:1839)
	at io.prestosql.jdbc.PrestoResultSet.getColumns(PrestoResultSet.java:1749)
	at io.prestosql.jdbc.PrestoResultSet.<init>(PrestoResultSet.java:118)
	at io.prestosql.jdbc.PrestoStatement.internalExecute(PrestoStatement.java:251)
	at io.prestosql.jdbc.PrestoStatement.execute(PrestoStatement.java:229)
	at org.apache.zeppelin.jdbc.JDBCInterpreter.executeSql(JDBCInterpreter.java:748)
	at org.apache.zeppelin.jdbc.JDBCInterpreter.interpret(JDBCInterpreter.java:831)
	at org.apache.zeppelin.interpreter.LazyOpenInterpreter.interpret(LazyOpenInterpreter.java:103)
	at org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer$InterpretJob.jobRun(RemoteInterpreterServer.java:632)
	at org.apache.zeppelin.scheduler.Job.run(Job.java:188)
	at org.apache.zeppelin.scheduler.FIFOScheduler$1.run(FIFOScheduler.java:140)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: io.prestosql.spi.PrestoException: [jcc][10165][10051][4.25.13] Invalid database URL syntax: jdbc:db2://db2whoc-flex-performance-riubnzs.services.dal.bluemix.net:50001/BLUDB:sslConnection=true. ERRORCODE=-4461, SQLSTATE=42815
	at io.prestosql.plugin.jdbc.BaseJdbcClient.getSchemaNames(BaseJdbcClient.java:132)
	at io.prestosql.plugin.jdbc.JdbcClient.schemaExists(JdbcClient.java:34)
	at io.prestosql.plugin.jdbc.JdbcMetadata.schemaExists(JdbcMetadata.java:67)
	at io.prestosql.metadata.MetadataManager.lambda$schemaExists$0(MetadataManager.java:290)
```

So it seems the plugin does not recognize the sslConnection option appended to the end of URL.

--- a/README.md
+++ b/README.md
@@ -11,6 +11,14 @@ Create new properties file inside `etc/catalog` dir:
     connection-user=myuser
     connection-password=mypassword
 
+For a connection with SSL, uses following JDBC URL strings as `connection-url`:
+
+    connection-url=jdbc:db2://ip:port/database:sslConnection=true;
+
+**Notice**: the trailing semi-colon is required. Or it will thrown SQLException `Invalid database URL syntax`.
+
+See official document of DB2 JDBC details from the article [Connecting programmatically with JDBC](https://www.ibm.com/support/knowledgecenter/en/SS6NHC/com.ibm.swg.im.dashdb.doc/connecting/connect_connecting_jdbc_applications.html).
+
 ## Building Presto DB2 JDBC Plugin
 
     mvn clean install

